posthog/__init__.py

```diff
@@ -406,8 +406,6 @@ def shutdown():
 def _proxy(method, *args, **kwargs):
     """Create an analytics client if one doesn't exist and send to it."""
     global default_client
-    if disabled:
```

Comment:
Breaking behaviour change here:

Earlier, if you did `posthog.capture()`, then `posthog.disabled = True`, then `posthog.capture()`, the second capture wouldn't fire, but now it would, since the initialised client has disabled = False.

Can do something like:

```
if disabled:
   default_client.disable()
else:
    default_client.enable()
```

and move it below the null check for `default_client`

Reply:
done